api/src/lists/daos/lists.dao.ts

```diff
@@ -113,8 +113,7 @@ export class ListsDao {
   }
```

Comment:
We should also change the name of the files to their singular form (`list.dao.ts`).